src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataSymbolMapExtensions.cs

```diff
@@ -206,5 +207,30 @@ public static bool IsSourceConstantArrayOfType(this TaintedDataSymbolMap<SourceI
 
             return taintedParameterPairs != null;
         }
+
+        private const string IndexerName = "this[]";
```

Comment:
Let's put this in a `TaintedDataProperties` internal static class, and also use it in `StringTransferSources`.